### PR TITLE
Added quiet mode on markdown link checker

### PR DESCRIPTION
**Pull request by Darshnadas**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -167,18 +167,22 @@ help:  ## Display this help
 .PHONY: test
 test: generate lint go-test ## Run generate lint and tests
 
+KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
```
Comment (CecileRobertMichon) [inline Makefile:170]: these changes shouldn't be in here, looks like you have an extra commit from your other PR here

Comment (Darshnadas) [reply Makefile:170]: Actually, I believe these changes are for another issue of this same project, this issue https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1990

Comment (Darshnadas) [reply Makefile:170]: I think I have raised the 2 PRs from the same branch, maybe that might be causing the issue?

Comment (Darshnadas) [reply Makefile:170]: How should I solve this?

**Comment by sayantani11** [comment]
@Darshnadas 
Try doing
`git rebase -i HEAD~2`
And drop the unnecessary commit, and push changes after the rebase

**Comment by Darshnadas** [comment]
okay @sayantani11

**Comment by mboersma** [comment]
@Darshnadas it's best if you can "force-push" (`git push -f`) to the same branch once you remove the unrelated changes, so a new PR isn't needed. But if that's not working out, you can always create a new PR. Thanks!

**Comment by Jont828** [comment]
@Darshnadas If you haven't fixed it yet, you can back up this branch, reset this branch to main, and cherry-pick the commit w/ your changes on to the clean branch and push. This is another option since you only have 2 commits. It would look like this.

```
git checkout lint_branch
git branch lit_branch_bak
git reset --hard origin/main # should be on lint_branch
git cherry-pick <commit hash of the one you want to keep>
git push -f # to Darshnadas:lint_check
```

File: Makefile (hunk 1 above)
Comment (Jont828) [reply Makefile:170]: If both PRs are based off of one branch then I think you might need to close and reopen one. For the PR you want to keep, i.e. this one, you can back up the branch, drop the extra commit, and force push. For the other PR, you can create a new branch off main and cherry pick the extra commit from your backed up branch (w/ both changes) to there.

**Comment by CecileRobertMichon** [comment]
Closing this in favor of #2156 

/close

**Comment by Darshnadas** [comment]
Okay sure, I'll run the above commands mentioned

**Comment by CecileRobertMichon** [comment]
@Darshnadas another PR was merged that fixes the issue so I went ahead and closed this one. Feel free to work on another help wanted issue (looks like you already have #2081 in progress as well).

**Comment by Darshnadas** [comment]
sure @CecileRobertMichon no issues

**Pull request closed without merging**